### DISCUSSION TITLE
[征集] 支持Mod方块 | Support mod blocks for SlopeCraft?

### DISCUSSION THREAD

**Discussion by ToKiNoBug** [SlopeCraft/SlopeCraft#120] [General｜大堂]
SlopeCraft经过爆改, 目前的架构已经允许添加原版之外的mod方块, 但我不清楚有哪些mod方块值得添加.

现在添加了夸克mod的一些半砖和树叶, 未来可以添加更多.

请告诉我有哪些mod的哪些方块值得添加.

SlopeCraft选用的方块应该是容易量产的完整方块, 尽量避免压力板这样必须依附其他方块的方块.

After huge midifications, SlopeCraft is currently able to support mod blocks. However, I'm unsure which mod blocks would be worth adding.

I've already added some slabs and leaves from the Quark mod, and I'm open to adding more in the future.

Please suggest some mod blocks that would be worthwhile to add.

**Comment by XU-12-song** [comment]
俺寻思可以添加机械动力
